src/gui/tray/ActivityItemContextMenu.qml

```diff
@@ -0,0 +1,65 @@
+import QtQml 2.15
+import QtQuick 2.15
+import QtQuick.Controls 2.3
+
+Item {
```

Comment:
Is there a reason why you are wrapping AutoSizingMenu inside an Item?

If you need to wrap it, you might want to use instead a Loader and unload the Menu completely to save a bit of RAM :)

Reply:
@CarlSchwan In fact, I need this wrapping `Item` as it was there before and it was put there in order to position the nested `Menu` properly. `Menu` does not allow some of the `anchors` properties, so, there was no other way to position the menu poperly.
And, as I already said - it was already there, so, just moving it into a separate `.qml`.

I've now added a `Loader` around `ActivityContextMenu.qml` into `ActivityItemActions.qml` if that helps. Using a `Loader` inside the `ActivityContextMenu.qml` doesn't seem to work, as, the property bindings to `moreActionsButtonContextMenu` stop working because the component is not loaded yet. Having said that, the menu is filled with entries and opened only if it is available (see the `Loader` I've added into `ActivityContextMenu.qml`).

Reply:
I am not sure (not tested) but from a quick reading of the code I would expect that indeed the `AutoSizingMenu` item could most probably replace the root `Item`

so that would mean moving all properties inside `root` `Item` item inside the `moreActionsButtonContextMenu` item

like this
```
AutoSizingMenu {
    id: moreActionsButtonContextMenu

    property int maxActionButtons: 0
    property var activityItemLinks: []
```

Reply:
@mgallien True, but, when I try doing that, the menu won't show up anymore